models.py

```diff
@@ -0,0 +1,151 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+
+
+
+class TimeStampedModel(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    isRemove = models.BooleanField(default=False)
+    class Meta:
+        abstract = True
+
+
+class Author(models.Model):
+    user = models.ForeignKey(User, related_name='author',db_index=True)
+    avatar = models.ImageField(upload_to='gallery/avatar/%Y/%m/%d',
+                               null=True,
+                               blank=True,
+                               help_text="Upload your photo for Avatar")
+    about = models.TextField()
+    website = models.URLField(max_length=200, blank=True, null=True)
+    isRemove = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.user.username
+
+    def get_absolute_url(self):
+        return reverse('author_posts_page',
+                       kwargs={'username': self.user.username})
+
+    class Meta:
+        verbose_name = 'Detail Author'
+        verbose_name_plural = 'Authors'

```

Comment:
We don't need these meta variables, as author plural is authors by default. Remove it please